packages/react-scripts/config/webpack.config.js

```diff
@@ -425,10 +425,13 @@ module.exports = function (webpackEnv) {
             },
             // load locale files
             {
-              test: /locales/,
-              loader: '@alienfast/i18next-loader',
+              test: /locales\/index\.js$/,
+              loader: isEnvProduction
+                ? require.resolve('../per-lang-loader')
```

Comment:
@redbugz it's currently pulled in here, though @tylergraf said he was changing that strategy to something else

Reply:
nvm, I see this is just getting index.js